CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Asking questions or requesting help
 
-Use the [Scylla Users mailing list](https://groups.google.com/g/scylladb-users) or the [Slack workspace](http://slack.scylladb.com) for general questions and help.
+Use the [ScyllaDB Community Forum](https://forum.scylladb.com) or the [Slack workspace](http://slack.scylladb.com) for general questions and help.
```

Comment:
Why replace instead of add?
Have we made an official decision to close the scylladb-users mailing list?

Reply:
We are not closing the mailing list, but the focus is the forum, and we prefer users to go there with questions (and answers)

Reply:
@nyh, the mailing list won't be closed at this point, but it won't be promoted either. We have already started moving announcements to the community forum and only posting a link to the thread from the mailing list.

Reply:
@avikivity if you agree with this direction, I'll you have the honor of merging it.

Reply:
@avikivity please have a look

Reply:
Thanks @avikivity , I fixed the metadata